android/src/play/java/com/dooboolab/RNIap/RNIapModule.kt

```diff
@@ -180,73 +181,72 @@ class RNIapModule(
         ) {
             val skuList = ArrayList<String>()
             for (i in 0 until skuArr.size()) {
-                val sku = skuArr.getString(i)
-                if (sku is String) {
+                if (skuArr.getType(i) == ReadableType.String) {
```

Comment:
Wouldn't `skuArr` always be the string array?

Reply:
Yes it should but ReadableArray holds Any type of objects. The old check for it to be string was redundant. In this way we guarantee that the elements are actually strings